server/controllers/Contacts.js

```diff
@@ -84,8 +85,13 @@ module.exports.getContacts = async (req, res) => {
         }
     });
 
-    if (!user) return res.status(404).send("user not found");
-    return res.send({"contactList": user.contactList});
+    if (!user) {
+        logger.error("User [" + userId + "] not found");
+        return res.status(404).send("user not found");
```

Comment:
update to match logger message? You should use msg variable to store message and also in .send({msg: msg})